sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 
 - Update live metrics to use typespec generated swagger
     ([#34840](https://github.com/Azure/azure-sdk-for-python/pull/34840))
+- Send old and new process level live metrics
```

Comment:
NIT: This makes it sound like we are sending both types rather than just the new type with both names